lib/features/family/features/reflect/domain/reflect_and_share_repository.dart

```diff
@@ -119,6 +119,16 @@ class ReflectAndShareRepository {
       role: const Role.sidekick(),
     );
 
+    if (_selectedProfiles.length == 2) {
```

Comment:
Just personal preference, but it's not really needed to write an 'else' as you just did a return

So just

```
if(..){
return x;
}

return x;
```
could be enough / cleaner

Reply:
Agreed, will fix